nifi-nar-bundles/nifi-hazelcast-bundle/nifi-hazelcast-services/src/main/resources/org.apache.nifi.hazelcast.services.cachemanager.StandaloneHazelcastCacheManager/additionalDetails.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>StandaloneHazelcastCacheManager</title>
+    <link rel="stylesheet" href="/nifi-docs/css/component-usage.css" type="text/css" />
+</head>
+
+<body>
+<h2>StandaloneHazelcastCacheManager</h2>
+
+<p>
+    This service connects to one or more existing Hazelcast instances as client. Hazelcast 4.0.0 or newer version is required.
+    The connection to the server is kept alive using the Hazelcast's built in reconnection capability. This might be fine-tuned
+    by setting the following properties:
```

Comment:
```suggestion
    This service connects to one or more existing Hazelcast instances as client. Hazelcast 4.0.0 or newer version is required.
    The connection to the server is kept alive using Hazelcast's built in reconnection capability. This might be fine-tuned
    by setting the following properties:
```